search/search_logger.go

```diff
@@ -13,16 +13,17 @@ import (
 )
 
 type SearchLog struct {
```

Comment:
Add to mapping:
https://github.com/Bnei-Baruch/archive-backend/blob/master/data/es/mappings/search_logs.json

```
"execution_time_log": {
  "type": "object", 
  "enabled": false
}, 
```

Reply:
Added